app/helpers/sessions_helper.rb

```diff
@@ -4,14 +4,35 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by id: user_id
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by id: user_id
+      if user&.authenticated?(cookies[:remember_token])
```

Comment:
```suggestion
      if user&.authenticated? cookies[:remember_token]
```
chỗ nào bỏ dc ngoặc thì bỏ e nhé